src/util.jl

```diff
@@ -33,3 +35,21 @@ function minimal_activity(ps::PresolveData{T}, i::Int)::T where {T}
     end
     return inf
 end
+
+# In some presolve procedures, when we process rows, we need to update
+# columns accordingly. This function performs synchronization between
+# rows and columns after a presolve method is applied on every rows.
+function col_sync(ps::PresolveData{T}) where {T}
+    i_index = zeros(Int, ps.ncol)
```

Comment:
Could you please add a comment describing what this array does and how it's used? Also, that it relies on the columns being sorted.